README.md

```diff
@@ -22,6 +22,7 @@ Stack Up is a simple deployment tool that performs given set of commands on mult
 | Option            | Description                      |
 |-------------------|----------------------------------|
 | `-f Supfile`      | Custom path to Supfile           |
+| `-i`, `sshKey`    | Set the the ssh key to use       |
```

Comment:
They call it `identity_file` in ssh command. I'm thinking if we should be consistent with them.